doc/source/user/troubleshooting-importerror.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
Is there somewhere we can link to this so its more discoverable?

Reply:
I'd prefer to leave this where it is, i.e. keep the `Setting Up` section on the front page of the docs. It's an important article and should be discoverable.